### PR TITLE
fix(ui): tidied up Alerts page design

### PR DESCRIPTION
**Pull request by robinrendle**
- Renamed Stream to Metric Alerts because...it’s just metric alert status for now
- Changed `Active` -> `Unresolved` 
- Removed the Beta badge
- Fixed the empty state copy for the metric alert list 

### Before
  
Screen Shot 2020-10-08 at 2 24 59 PM

### After

Screen Shot 2020-10-08 at 2 12 30 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/views/alerts/list/header.tsx [hunk 1]
```diff
@@ -36,28 +35,18 @@ const AlertHeader = ({router, organization, activeTab}: Props) => {
...
-          <FeatureBadge
-            title={
-              activeTab === 'stream'
-                ? t('This page is in beta and currently only shows metric alerts.')
-                : undefined
-            }
-            type="beta"
-          />
-        </StyledLayoutTitle>
+        <StyledLayoutTitle>{t('Alerts')} </StyledLayoutTitle>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/alerts/list/header.tsx:38]:
```suggestion
        <StyledLayoutTitle>{t('Alerts')}</StyledLayoutTitle>
```

File: src/sentry/static/sentry/app/views/alerts/list/index.tsx [hunk 2]
```diff
@@ -265,22 +263,24 @@ class IncidentsList extends AsyncComponent<Props, State & AsyncComponent['state'
...
-                  </Button>
-                  <Button
-                    to={{pathname, query: closedIncidentsQuery}}
-                    barId="closed"
-                    size="small"
-                  >
-                    {t('Resolved')}
-                  </Button>
-                </StyledButtonBar>
+                <NavWrapper>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/alerts/list/index.tsx:266]: Is this needed? There is only one child

File: src/sentry/static/sentry/app/views/alerts/list/header.tsx (hunk 1 above)
Comment (robinrendle) [reply src/sentry/static/sentry/app/views/alerts/list/header.tsx]: Gah, fixed.

File: src/sentry/static/sentry/app/views/alerts/list/index.tsx (hunk 2 above)
Comment (robinrendle) [reply src/sentry/static/sentry/app/views/alerts/list/index.tsx]: Nope, this is leftover cruft from what I was fooling around with.

**Review by scttcper — APPROVED** [review]
🎨

**Pull request merged**